common/test/acceptance/tests/lms/test_lms_edxnotes.py

```diff
@@ -11,7 +10,6 @@
 from ...fixtures.edxnotes import EdxNotesFixture, Note, Range
 
 
-@skipUnless(os.environ.get("FEATURE_EDXNOTES"), "Requires Student Notes feature to be enabled")
```

Comment:
This is the only place in all of edx-platform where "FEATURE_EDXNOTES" appears. So I don't know where it came from... perhaps the environment variable was later renamed to ENABLE_EDXNOTES?